akka-cluster/src/main/scala/akka/cluster/ClusterReadView.scala

```diff
@@ -45,17 +48,16 @@ private[akka] class ClusterReadView(cluster: Cluster) extends Closeable {
   @volatile
   private var _latestStats = CurrentInternalStats(GossipStats(), VectorClockStats())
 
-  val selfAddress = cluster.selfAddress
+  val selfAddress: Address = cluster.selfAddress
 
   // create actor that subscribes to the cluster eventBus to update current read view state
   private val eventBusListener: ActorRef = {
     cluster.system
       .systemActorOf(Props(new Actor with RequiresMessageQueue[UnboundedMessageQueueSemantics] {
         override def preStart(): Unit = cluster.subscribe(self, classOf[ClusterDomainEvent])
-        override def postStop(): Unit = cluster.unsubscribe(self)
```

Comment:
unsubscribe from eventBus is anyway automatic and this will most likely end in deadLetters anyway since cluster actors are stopped in the shutdown process